### DISCUSSION TITLE
Version name V2.0

### DISCUSSION THREAD

**Discussion by AdriaGual** [AdriaGual/pokemon-pocket-bot#23] [Announcements]
With the recent refactor by @NomadCHK, I think it's time to consider this a new version. I named the first version "Amiable Azurill" (V1.0.0) since it was our initial release, and now it's time for V2!

For this new version, I’m looking for ideas for a name, following the pattern of an adjective and a Pokémon, both starting with the letter B. Here are a few suggestions I have:

- Brave Bulbasaur
- Bouncy Breloom
- Bold Bisharp

I’d love to hear your thoughts and any other suggestions you might have. The name that resonates the most with us will be the official version name!

**Comment by NomadCHK** [comment]
Hey, that's a very nice idea! Let me suggest something... do you know what a "Boto" is? 
If you search for "boto pescadores" on Google, you'll see it's a freshwater dolphin from the Amazon. 
Boto is also my nickname, and while it's not a Pokémon, it's a special animal here in my region of Brazil that starts with B. 
Interestingly, in Brazilian culture, "boto" is also used as a nickname for people who are helpful and kind to others - someone who's always willing to lend a hand!

There's also a fascinating Amazon legend about the Boto: according to folklore, it transforms into a handsome young man during festivals, wearing white clothes and a hat (to hide its blowhole). It charms young women with its charisma and dancing skills before returning to the river at dawn. Even though I live in the South of Brazil, far from the Amazon, this legend is known throughout our country! 

Actually, in my region, we have our own special connection with dolphins (botos)! They help the fishermen at the river mouth where it meets the sea, guiding them to find fish. It's a beautiful example of cooperation between humans and nature.

Maybe we could find a way to incorporate this idea? 😊

**Comment by AdriaGual** [comment]
That works perfectly for me; we just need to check for an adjective.

"Bold" could be a great choice, like Bold Boto—it's short and easy to call, and I like that one.

Let's keep the discussion open for a few more days in case others have more ideas, but I'm happy with this one!

Here's a little preview of the idea:

<a href="https://tcgpocket.Pokemon.com/es-es/"><img src="https://github.com/user-attachments/assets/2419abaa-4a0b-4f53-83bc-eadf2199ee2e" alt="Markdownify" width="200"></a>

**Comment by NomadCHK** [comment]
Hi AdriaGual and everyone!

I’d like to propose Boto Binacle for V2.0. Here’s why:

    Boto: Represents kindness and helpfulness, aligning with our bot’s mission.
    Binacle: Has two hands, symbolizing support, and its evolution, Barbaracle, has more hands, representing our growth and added features in V2.0.

Boto Binacle captures both the cultural significance and the functional improvements we’re bringing. What do you think?

Looking forward to your thoughts! 😊
